tests/unit/core/reducers/test_addons.js

```diff
@@ -105,4 +112,72 @@ describe('addon reducer', () => {
       baz: { description: null },
     });
   });
+
+  it('exposes "isRestartRequired" attribute from current version files', () => {
+    const addon = createFakeAddon({
+      files: [
+        { ...fakeAddon.current_version.files[0], is_restart_required: true },
+      ],
+    });
+
+    expect(
+      addons(undefined, loadEntities(createFetchAddonResult(addon).entities))
+    ).toEqual({
+      [addon.slug]: denormalizeAddon({
+        ...addon,
+        installURL: '',
+        isRestartRequired: true,
+      }),
+    });
+  });
+
+  it('sets "isRestartRequired" to "false" when addon does not need restart', () => {
```

Comment:
This is a near-OCD nitpick but I like the markdown convention of using backticks for things that are "code". So I'd rather see things like object properties and `false` written with <code>\`</code> instead of `"` around them. Wouldn't r- thing, but I'd probably change it next time I edited the file :wink: